cmd/collector/app/collector.go

```diff
@@ -103,24 +99,24 @@ func (c *Collector) Start(options *flags.CollectorOptions) error {
 	c.spanHandlers = handlerBuilder.BuildHandlers(c.spanProcessor)
 
 	grpcServer, err := server.StartGRPCServer(&server.GRPCServerParams{
```

Comment:
I would suggest changing server.GRPCServerParams to directly embed configgrpc struct, so that you don't have to do any field copying here